thunder/torch/__init__.py

```diff
@@ -1812,6 +1812,25 @@ def leaky_relu(a: TensorProxy, /, negative_slope: float = 0.01, inplace: bool =
 _inplace_to_out_of_place[leaky_relu] = leaky_relu, 2
 
 
+@torchsymbol(torch.nn.functional.logsigmoid, is_method=False)
+def logsigmoid(a: TensorProxy, /) -> TensorLike:
+    return where(a > 0, -log1p(exp(-a)), a - log1p(exp(a)))
+
+
+_inplace_to_out_of_place[logsigmoid] = logsigmoid, -1
```

Comment:
At glance logsigmoid doesn't seem to store the results into the inout tensor. Is where an in-place op?

Reply:
Then I don't think this registration is necessary.

The second value is the index of a boolean arg indicating the op is inplace or not, if exists. So the comment here is misleading, looking at how it's used https://github.com/Lightning-AI/lightning-thunder/blob/d59f9ab16b8e7dd797e3c9849a72653634dd7142/thunder/core/functionalization.py#L320-L326
https://github.com/Lightning-AI/lightning-thunder/blob/d59f9ab16b8e7dd797e3c9849a72653634dd7142/thunder/torch/__init__.py#L94-L95

Reply:
ok, updated